docs/source/optimization/algorithms.rst

```diff
@@ -0,0 +1,78 @@
+.. _list_of_algorithms:
+
+algo_options
+============
+
+
+``algo_options`` is a dictionary with optional keyword arguments that are passed
+to the optimizer. This includes tolerances for the termination criteria,
+parameters that determine how greedy the optimizer how large the stepsize for
```

Comment:
Maybe explain what you mean with "how greedy the optimizer". Also there is av verb missing I think.

Reply:
I don't want to add more explanations. This is just one example and there are two more, so people who don't understand it can continue.